src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/InstanceClient.cs

```diff
@@ -96,7 +96,7 @@ public async Task<Instance> GetInstance(Instance instance)
         /// <inheritdoc />
         public async Task<List<Instance>> GetInstances(Dictionary<string, StringValues> queryParams)
         {
-            StringBuilder apiUrl = new($"instances?");
+            StringBuilder apiUrl = new ($"instances?");
```

Comment:
no space before new when using type inference, although both vs and sonarcloud mark it as a code smell.

Reply:
Hm, I think we should stick with the rules/hints from vs/sonarcloud - too much brain power wasted to consider if we should fix issues like this or not. Let the tools decide. Either that, or we should change the rules, but probably not worth the effort - I trust those that created the rules in vs/sonarcloud more than I trust my self :)

Reply:
We already have several team members holding onto to the opposite, so I would suggest to run it past the team so we have the same practice.

Reply:
It's perfectly fine if we decide that we don't like the rules provided by the tooling, but then I really think those arguing against the rules provided, and decided in the first place that they don't won't to follow those rules, should change the rule definitions in the tooling. Otherwise it will be hopeless for the rest of us to know what's expected and we end up wasting time in discussions like this.